server/src/test/java/com/broadcom/lsp/cobol/service/delegates/completions/CompletionsChainTest.java

```diff
@@ -65,11 +65,11 @@
           + "       200-Get-Date. A\n"
           + "       300-Say-Goodbye.\n"
           + "       End program ProgramId.";
+  @Inject private TextDocumentService service;
+  @Inject private TestLanguageClient client;
 
   @BeforeEach
   void createService() {
-    service = injector.getInstance(TextDocumentService.class);
-    TestLanguageClient client = injector.getInstance(TestLanguageClient.class);
     client.clean();
     runTextValidation(service, TEXT);
     waitForDiagnostics(client);
```

Comment:
This test should work directly with the `Completions` class, providing a predefined model to retrieve completion suggestions.